doc/Type/List.pod6

```diff
@@ -919,25 +935,49 @@ for operators which aren't left-associative:
     ((2-3)-4).say;                        # OUTPUT: «-5␤»
     (2-3-4).say;                          # OUTPUT: «-5␤»
 
-Since reducing with an infix operator is a common thing to do, the C<[ ]>
-metaoperator provides a syntactic shortcut:
+Since reducing with an infix operator is a common thing to do, the
+L<reduction metaoperator|/language/operators#Reduction_metaoperators> C<[ ]>
+provides a syntactic shortcut. Thus, instead of passing the operator's code
+object to C<reduce>, just pass the operator directly to C<[ ]>. To use a
+user-defined subroutine instead, provide an additional layer of square brackets
+around the subroutine's code object:
 
-    # The following all do the same thing...
-    my @numbers = (1,2,3,4,5);
+    say [*] [1, 2, 3, 4];       # OUTPUT: «24␤»
+    say [min] [4, 2, 1, 3];     # OUTPUT: «1␤»
+
+    sub mult { $^a * $^b };
+    say [[&mult]] [1, 2, 3, 4]; # OUTPUT: «24␤»
+
+Semantically, all the following do the same thing:
+
+    my @numbers = 1, 2, 3, 4, 5;
     say reduce { $^a + $^b }, 0, |@numbers;
     say reduce * + *, 0, |@numbers;
-    say reduce &[+], @numbers; # operator does not need explicit identity
-    say [+] @numbers;          # most people write it this way
+    say reduce &[+], @numbers; # operator does not need explicit identity value
+    say [+] @numbers;
+
+Since C<reduce> is an implicit loop, it responds to C<next>, C<last> and C<redo>
```

Comment:
"is an" was already there, but again, maybe express it better. "Iterates over its reducing function?"